service/templates/admin.html

```diff
@@ -133,6 +133,17 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 											</tr>
 										</thead>
 										<tbody>
+											<tr>
+												<td>Plugin settings on admin page</td>
```

Comment:
Another person looking at this may be confused about what it means, since the plugin settings will be visible on the admin page regardless of this toggle. Maybe "Use admin-level plugin settings" or etc?